mlrun/projects/pipelines.py

```diff
@@ -73,7 +73,7 @@ def __init__(
         ttl=None,
         args_schema: dict = None,
         schedule: typing.Union[str, mlrun.api.schemas.ScheduleCronTrigger] = None,
-        overwrite: bool = None,
+        override: bool = None,
```

Comment:
this breaks backwards compatibility if users used that straight from the SDK... need to discuss

Reply:
Users don't use that, as far as I know the user should use `project.set_workflow()`